datapacks/Bookshelf/data/bs.block/tags/function/fill_random.json

```diff
@@ -0,0 +1,20 @@
+{
+  "__bookshelf__": {
+    "feature": true,
+    "documentation": "https://bookshelf.docs.gunivers.net/en/latest/modules/block.html#fill",
+    "authors": [
+      "Aksiome"
+    ],
```

Comment:
According to what we discuss, I think we should add [SBtree-bit](https://github.com/SBtree-bit) as contributor since they give the idea.